common/src/test/java/com/hyperwallet/android/common/util/DateUtilityTest.java

```diff
@@ -0,0 +1,56 @@
+package com.hyperwallet.android.common.util;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.Test;
+
+import java.util.Date;
+
+public class DateUtilityTest {
```

Comment:
just being a little picky here... but use returns for all test names instead of return, ie. testToDateFormat_returnExpectedStringFormat would change to testToDateFormat_returnsExpectedStringFormat